fastai/data_block.py

```diff
@@ -121,10 +121,11 @@ def from_df(cls, df:DataFrame, path:PathOrStr='.', cols:IntsOrStrs=0, processor:
         return res
 
     @classmethod
-    def from_csv(cls, path:PathOrStr, csv_name:str, cols:IntsOrStrs=0, header:str='infer', 
-                 processor:PreProcessors=None, **kwargs)->'ItemList':
-        "Create an `ItemList` in `path` from the inputs in the `cols` of `path/csv_name` opened with `header`."
-        df = pd.read_csv(Path(path)/csv_name, header=header)
+    def from_csv(cls, path:PathOrStr, csv_name:str, cols:IntsOrStrs=0, delimiter:str=None, header:str='infer', 
+                 col_names:Optional[Collection[str]]=None, processor:PreProcessors=None, **kwargs)->'ItemList':
+        """Create an `ItemList` in `path` from the inputs in the `cols` of `path/csv_name` opened with `header`. 
```

Comment:
Let's leave it a the first line (fastai style guide) and the second line of this doc string should go in the doc notebook).